src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -7,7 +7,8 @@ public class Messages {
 
     public static final String MESSAGE_UNKNOWN_COMMAND = "Unknown command";
     public static final String MESSAGE_INVALID_COMMAND_FORMAT = "Invalid command format! \n%1$s";
-    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid";
+    public static final String MESSAGE_INVALID_PERSON_DISPLAYED_INDEX = "The person index provided is invalid!";
```

Comment:
Remove the `!`, as we're not allowed to "enhance" the error message.

But adding `MESSAGE_DUPLICATE_PERSONS_INDEX` should be fine, since its a bug.